holmes.go

```diff
@@ -159,7 +160,9 @@ func (h *Holmes) startGCCycleLoop() {
 
 // Start starts the dump loop of holmes.
 func (h *Holmes) Start() {
-	atomic.StoreInt64(&h.stopped, 0)
+	if !atomic.CompareAndSwapInt64(&h.stopped, 1, 0) {
+		return
```

Comment:
maybe it's better to add a long when restarting it again?

Reply:
I don't quite understand.
Add a restart?

Reply:
add a log before return since start twice should be unexpected.
the log is just a warning for the caller.

Reply:
OK, I think holmes can be configured to start externally, and repeated booting may be expected.the output log is also correct.